cadence/transactions/evm/create_new_account_with_coa.cdc

```diff
@@ -0,0 +1,64 @@
+import Crypto
+
+import "EVM"
+
+/// Creates a new Flow Address and EVM account with a Cadence Owned Account (COA) stored in the account's storage.
```

Comment:
We could precise that the transaction creates an account with a single full-weight key